java/servlets/security/cookie-issecure-false.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: cookie-issecure-false
+  patterns:
+  - pattern-either:
```

Comment:
Nit: we don't need this `pattern-either`. It makes it a little confusing as to which patterns are being `AND`'ed.